internal/dcache/replication_manager/replication_manager.go

```diff
@@ -498,7 +498,7 @@ func syncMV(mvName string, mvInfo dcache.MirroredVolume) {
 	// TODO: Make sure GetDiskUsageOfMV() correctly returns the to-be-synced data, i.e., data in the regular
 	//       MV folder.
 	//
-	syncSize, err := rpc_server.GetDiskUsageOfMV(mvName, lioRV)
+	syncSize, err := rpc_server.GetMyMVSize(mvName, lioRV)
```

Comment:
let's not have a special GeyMyMVSize() method exposed by rpc server for MVs hosted by us, instead just define one single function exported by rpc package.

func GetMVSize(mvName string) (int64, error)

then you can use this same function inside fixMV() and here.

Note that GetMVSize() is more like ReadMV() where the target of the operation is MV but any of the RVs is capable of serving that, just like ReadMV(), GetMVSize() should pick one at random (favoring the local one).

Also, fix the fixMV() in this same change as it's just that function we need to call.

Reply:
Update the GetMVSize() method, please review